### PR TITLE
use the shiny new signing server (bug 1115184)

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: mkt/settings.py [hunk 1]
```diff
@@ -1077,11 +1077,11 @@ def JINJA_CONFIG():
 SIGNED_APPS_OMIT_PER_FILE_SIGS = True
 
 # This is the signing REST server for signing receipts.
-SIGNING_SERVER = ''
+SIGNING_SERVER = (os.environ.get('SIGNING_SERVER', ''))
```
Comment (robhudson) [inline mkt/settings.py]: Wrapping parens not needed.

**Comment by robhudson** [comment]
I discovered https://pypi.python.org/pypi/python-decouple/2.3 recently and I like it. We should switch all of our `os.environment` calls to it. It would allow for defaults and proper casting, as well as other options for where to store these.

File: mkt/settings.py (hunk 1 above)
Comment (andymckay) [inline mkt/settings.py]: doh, thanks

**Comment by robhudson** [comment]
r+. python-decouple is still cool.

**Comment by andymckay-limited-access** [comment]
I am more than happy for you to use python-decouple in zamboni :)

**Pull request merged**